src/test/resources/implementationNotes.json

```diff
@@ -55,11 +51,11 @@
             "controllerName":"activity-controller",
             "endPointLink":[  
                {  
-                  "implementationNote":"ROLE_ADMIN AND ROLE_ONC_ADMIN may set the 'showDeleted' flag to true and see activity for all certification bodies including that have been deleted. ROLE_ACB can see their own information.",
+                  "implementationNote":"ROLE_ADMIN AND ROLE_ONC may set the 'showDeleted' flag to true and see activity for all certification bodies including that have been deleted. ROLE_ACB can see their own information.",
```

Comment:
I don't think `showDeleted` exists any more. It might have in the Excel in the beginning, but I believe it's since been removed with the implementation of ACB/ATL retirement